themes/V3/Journal/snippets.js

```diff
@@ -0,0 +1,408 @@
+/* eslint-disable max-lines */
+
+const MagicGen           = require('./snippets/magic.gen.js');
+const ClassTableGen      = require('./snippets/classtable.gen.js');
+const MonsterBlockGen    = require('./snippets/monsterblock.gen.js');
+const ClassFeatureGen    = require('./snippets/classfeature.gen.js');
+const CoverPageGen       = require('./snippets/coverpage.gen.js');
+const TableOfContentsGen = require('./snippets/tableOfContents.gen.js');
+const WatercolorGen      = require('./snippets/watercolor.gen.js');
```

Comment:
These files don't exist. Would there be benefit in a `../../commonSnippets/` location?

Reply:
Yes, I'm actually creating a "blank" theme that has just the common, theme-agnostic snippets (insert page break, insert image, insert basic table, etc.) This "blank" theme will always be loaded no matter what theme you choose. This way every theme doesn't have to build from the PHB as a base, but they can at least have the bare minimum needed to work with v3 syntax already working.